README.md

```diff
@@ -1,86 +1,86 @@
-
-    <meta charset="utf-8"/>
-    <table>
-    <tbody>
-        <tr>
-            <td align="left">EmbedSecret</td>
-            <td align="left">Get your EmbedSecret key from the Embed tab by enabling the `Enable embed authentication` on the Administration page https://help.boldbi.com/embedded-bi/site-administration/embed-settings/.</td>
-        </tr>
-        <tr>
-            <td align="left">UserEmail</td>
-            <td align="left">UserEmail of the Admin in your Bold BI server or have permission to view the dashbaord, which would be used to render the dashboard.</td>
-        </tr>
-    </tbody>
-    </table>
-	
-  * Open the `index.html` file in NodeJs sample and provide the value for following properties,
+> **NOTE:** Node.js versions 12.13 to 18.16 are supported.
+
+ #### Help link
+
+ * https://help.boldbi.com/embedded-bi/faq/where-can-i-find-the-product-version/
+
+ #### Supported browsers
   
-    <meta charset="utf-8"/>
-        <table>
-        <tbody>
-            <tr>
-                <td align="left">RootUrl</td>
-                <td align="left">Dashboard Server URL (Eg: http://localhost:5000/bi, http://demo.boldbi.com/bi).</td>
-            </tr>
-            <tr>
-                <td align="left">SiteIdentifier</td>
-                <td align="left">For the Bold BI Enterprise edition, it should be like `site/site1`. For Bold BI Cloud, it should be an empty string.</td>
-            </tr>
-            <tr>
-                <td align="left">Environment</td>
-                <td align="left">Your Bold BI application environment. (If Cloud, you should use `cloud,` if Enterprise, you should use `enterprise`).</td>
-            </tr>
-            <tr>
-                <td align="left">dashboardId</td>
-                <td align="left">Provide the dashboard id of the dashboard you want to embed in view or edit mode. Ignore this property to create new dashboard.</td>
-            </tr>
-            <tr>
-                <td align="left">authorizationUrl</td>
-                <td align="left">Url of the 'GetDetails' action in the ASP.NET application.</td>
-            </tr>
-        </tbody>
-        </table>
-
-### Install npm
-
-To install all dependent packages, use the below command
-
-```bash
-npm install
-```
-### Run/Serve
-
-To run the samples, use the below command
-
-```bash
-node embed.js
-```
-
-Please refer to the [help documentation](https://help.boldbi.com/embedded-bi/javascript-based/samples/v3.3.40-or-later/node-js/#how-to-run-the-sample) to know how to run the sample.
-
-## Online Demos
+  * Google Chrome, Microsoft Edge, Mozilla Firefox, and Safari.
+
+ ## Configuration
+
+ * Please ensure that you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.
```

Comment:
* Please ensure you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/site-administration/embed-settings/#get-embed-secret-code) to enable it.

Reply:
Content updated